### DISCUSSION TITLE
addling link to img tags

### DISCUSSION THREAD

**Discussion by Bayramito** [ueberdosis/tiptap#5963] [General Questions]
Can we make img tags clickable?

Can we wrap them up with a tags ?

Why this is not a built in feature ?

**Comment by guanriyue** [comment]
Wrapping an image with an `a` tag is certainly possible. It’s important to note that whether a node supports a mark is not determined by the node itself but rather by whether its parent node supports the mark. 

If your image is an inline node, the default paragraph element supports the `link` mark, so you can use an `a` tag to wrap the image. 

If your image is a block node, it's better to set up a specific parent node for the image, such as `imageFigure`, and ensure that `imageFigure` supports the `link` mark.
